ytyp/properties/flags.py

```diff
@@ -129,133 +129,83 @@ class EntityFlags(FlagPropertyGroup, bpy.types.PropertyGroup):
     flag17: bpy.props.BoolProperty(
         name="Double-sided rendering", update=FlagPropertyGroup.update_flag)
     flag18: bpy.props.BoolProperty(
         name="Dynamic", update=FlagPropertyGroup.update_flag)
     flag19: bpy.props.BoolProperty(
-        name="Unknown 19", update=FlagPropertyGroup.update_flag)
+        name="Override Physics Bounds", update=FlagPropertyGroup.update_flag)
     flag20: bpy.props.BoolProperty(
-        name="Dynamic anims (YCD)", update=FlagPropertyGroup.update_flag)
+        name="Auto Start Anim", update=FlagPropertyGroup.update_flag)
     flag21: bpy.props.BoolProperty(
-        name="Unknown 21", update=FlagPropertyGroup.update_flag)
+        name="Has Pre Reflected Water Proxy", update=FlagPropertyGroup.update_flag)
     flag22: bpy.props.BoolProperty(
-        name="Unknown 22", update=FlagPropertyGroup.update_flag)
+        name="Has Drawable Proxy For Water Reflections", update=FlagPropertyGroup.update_flag)
     flag23: bpy.props.BoolProperty(
-        name="Unknown 23", update=FlagPropertyGroup.update_flag)
+        name="Does Not Provide AI Cover", update=FlagPropertyGroup.update_flag)
     flag24: bpy.props.BoolProperty(
-        name="Unknown 24", update=FlagPropertyGroup.update_flag)
+        name="Does Not Provide Player Cover", update=FlagPropertyGroup.update_flag)
     flag25: bpy.props.BoolProperty(
-        name="Unknown 25", update=FlagPropertyGroup.update_flag)
+        name="Is Ladder Deprecated", update=FlagPropertyGroup.update_flag)
     flag26: bpy.props.BoolProperty(
-        name="Unknown 26", update=FlagPropertyGroup.update_flag)
+        name="Has Cloth", update=FlagPropertyGroup.update_flag)
     flag27: bpy.props.BoolProperty(
-        name="Enables special attribute", update=FlagPropertyGroup.update_flag)
+        name="Enable Door Physics", update=FlagPropertyGroup.update_flag)
     flag28: bpy.props.BoolProperty(
-        name="Unknown 28", update=FlagPropertyGroup.update_flag)
+        name="Is Fixed For Navigation", update=FlagPropertyGroup.update_flag)
     flag29: bpy.props.BoolProperty(
-        name="Disable red vertex channel", update=FlagPropertyGroup.update_flag)
+        name="Dont Avoid By Peds", update=FlagPropertyGroup.update_flag)
     flag30: bpy.props.BoolProperty(
-        name="Disable green vertex channel", update=FlagPropertyGroup.update_flag)
+        name="Use Ambient Scale", update=FlagPropertyGroup.update_flag)
     flag31: bpy.props.BoolProperty(
-        name="Disable blue vertex channel", update=FlagPropertyGroup.update_flag)
+        name="Is Debug", update=FlagPropertyGroup.update_flag)
     flag32: bpy.props.BoolProperty(
-        name="Disable alpha vertex channel", update=FlagPropertyGroup.update_flag)
+        name="Has Alpha Shadow", update=FlagPropertyGroup.update_flag)
 
 
-class UnknownFlags(FlagPropertyGroup, bpy.types.PropertyGroup):
+class MloFlags(FlagPropertyGroup, bpy.types.PropertyGroup):
```

Comment:
This flags are wrong, they don't start at bit 0.

Also need to add `size = 7` since it no longer matches the default size of 32.